### ISSUE TITLE
List widget keyboard accessibility needs to be on par with tree

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#8621]
When using the extensions viewlet with the keyboard you notice some differences to how it works with the tree. If you tab into the list, the entire list gets the focus outline, where actually the currently selected item should just highlight.

Also, when the list widget has focus, pressing arrow key down should navigate through the list as it does in the tree.

Whatever magic we did in the tree to make it work nice with keyboard also needs to be applied to the list widget 👍 

Related https://github.com/Microsoft/vscode/issues/7966

**Comment by joaomoreno** [comment]
Agreed!!!

**Comment by stereokai** [comment]
I think this issue is not fixed because VSCode is now missing the ability to remove an extension via the keyboard. (Or otherwise I'm missing it).

**Comment by joaomoreno** [comment]
This is about keyboard accessibility of the extensions view, not about creating keybindings for installing/uninstalling extensions.

**Comment by stereokai** [comment]
Okay. I'll focus on the other thread.

**Comment by jrieken** [comment]
@bpasero @joaomoreno unsure what the desired behaviour is but the explorer is the only that behaves different wrt focusing the content area of the tree vs. immediately focusing a tree item

**Comment by bpasero** [comment]
@jrieken can you be more specific? I can cause the explorer to focus nothing by clicking into empty space.

**Comment by jrieken** [comment]
I would see a pick focus border on the tree (despite it having children) Tho not seeing that anymore...
